CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Default missing `Event.platform` and `Event.level` fields during light normalization. ([#2961](https://github.com/getsentry/relay/pull/2961))
 - Copy event measurements to span & normalize span measurements. ([#2953](https://github.com/getsentry/relay/pull/2953))
 - Add possiblity to block metrics or their tags with glob-patterns. ([#2954](https://github.com/getsentry/relay/pull/2954), [#2973](https://github.com/getsentry/relay/pull/2973))
+- Enable periodic unspool of the buffered envelopes. ([#2993](https://github.com/getsentry/relay/pull/2993))
```

Comment:
If I understand correctly, the main purpose of this PR is to not unspool everything at once, right?

```suggestion
- Enable throttled periodic unspool of the buffered envelopes. ([#2993](https://github.com/getsentry/relay/pull/2993))
```